common/jmeter/setup/Post_image.jmx

```diff
@@ -0,0 +1,298 @@
+            <stringProp name="Argument.name">usernamePrefix</stringProp>
+            <stringProp name="Argument.value">isTestUser_</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="password" elementType="Argument">
+            <stringProp name="Argument.name">password</stringProp>
+            <stringProp name="Argument.value">password</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="rolename" elementType="Argument">
+            <stringProp name="Argument.name">rolename</stringProp>
+            <stringProp name="Argument.value">isTestUserRole</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+        </collectionProp>
+        <stringProp name="TestPlan.comments">ec2-34-207-59-213.compute-1.amazonaws.com</stringProp>
+      </Arguments>
+      <hashTree/>
+      <Arguments guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
+        <collectionProp name="Arguments.arguments">
+          <elementProp name="noOfThreads" elementType="Argument">
+            <stringProp name="Argument.name">noOfThreads</stringProp>
+            <stringProp name="Argument.value">${__P(concurrency,1)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="noOfImages" elementType="Argument">
+            <stringProp name="Argument.name">noOfImages</stringProp>
+            <stringProp name="Argument.value">${__P(imageCount,1000)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="loopCount" elementType="Argument">
+            <stringProp name="Argument.name">loopCount</stringProp>
+            <stringProp name="Argument.value">${__P(loopCount,1000)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="rampUpPeriod" elementType="Argument">
+            <stringProp name="Argument.name">rampUpPeriod</stringProp>
+            <stringProp name="Argument.value">${__P(rampUpPeriod,10)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="locationCsvPath" elementType="Argument">
+            <stringProp name="Argument.name">locationCsvPath</stringProp>
+            <stringProp name="Argument.value">/home/ubuntu/location.csv</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="imageLocationPath" elementType="Argument">
+            <stringProp name="Argument.name">imageLocationPath</stringProp>
+            <stringProp name="Argument.value">/home/ubuntu/image.png</stringProp>
```

Comment:
Is this path correct?

Reply:
yes path is correct.

Reply:
I noticed that you've uploaded an image as well. Can't we use that? User will have to save an image to /home/ubuntu for this to work right?

Reply:
Image file already given, [1].

[1] https://github.com/wso2/performance-is/pull/63/files#diff-c68999ca672d9d5510ec301aab718938

Reply:
Yes, my question is, the path /home/ubuntu/image.png is correct or not. I see that you've added the image into common/lib folder but in the script we refer it from /home/ubuntu?

Reply:
Image file will be place on the bastion node at the /home/ubuntu path from the set up file.